src/lib.rs

```diff
@@ -603,6 +615,45 @@ impl LightningNode {
         get_payment_max_routing_fee_mode(amount_sat, &self.get_exchange_rate())
     }
 
+    /// Get the affordability status of an invoice. Throws an error if the invoice doesn't have a
+    /// requested amount.
```

Comment:
Should we change the method to also allow affordability checks for open invoices? Either allow an optional amount to also be provided, or even change the method to only take a sat amount (seems like we only care about the amount in the invoice).

Reply:
If we take only a sat amount, it would be useful not only for paying bolt11 invoices, but also LNURL and LN addresses.